students/AndrewMiotke/session04/reading_and_parsing.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# TODO:
+#  - create two lists
+#     - list one: person's name
+#     - list two: language
+# - zip the two lists together
+
+
+def open_file(file_name):
+    languages = []
+    people = []
+
+    with open(file_name, "r") as input_file:
+        read_file = input_file.readlines()
+
+        for line in read_file:
```

Comment:
instead of  calling `readlines()` and then looping, you can simply loop through the lines of the file directly:

```
for line in input_file:
   ...
```